Hudl.Mjolnir/Command/Command.cs

```diff
@@ -380,9 +380,17 @@ private Task<TResult> ExecuteInIsolation(CancellationToken cancellationToken)
                 CircuitBreaker.MarkSuccess(stopwatch.ElapsedMilliseconds);
                 CircuitBreaker.Metrics.MarkCommandSuccess();
             }
-            catch (Exception)
+            catch (Exception e)
             {
-                CircuitBreaker.Metrics.MarkCommandFailure();
+                if (CommandContext.IsExceptionIgnored(e.GetType()))
```

Comment:
Yeah, possibly. I'm not sure how expensive it'll be to add an `IsAssignableFrom()` or equivalent call into here, though. I'll look into it - good call.